applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -29,38 +29,53 @@
+				    1, _stream_context),                                           \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(                                                      \
+			LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE_SINK), 2u, 20u,           \
+			CONFIG_BT_AUDIO_PRESENTATION_DELAY_US))
+
+#define BT_AUDIO_LC3_UNICAST_PRESET_SOURCE_RECOMMENDED(_loc, _stream_context)                      \
 	BT_AUDIO_LC3_PRESET(                                                                       \
 		BT_CODEC_LC3_CONFIG(BT_AUDIO_CODEC_CONFIG_FREQ, BT_CODEC_CONFIG_LC3_DURATION_10,   \
-				    _loc, LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE), 1,    \
+				    _loc,                                                          \
+				    LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE_SOURCE), 1,   \
 				    _stream_context),                                              \
-		BT_CODEC_LC3_QOS_10_UNFRAMED(LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE),    \
-					     2u, 20u, CONFIG_BT_AUDIO_PRESENTATION_DELAY_US))
+		BT_CODEC_LC3_QOS_10_UNFRAMED(                                                      \
+			LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE_SOURCE), 2u, 20u,         \
+			CONFIG_BT_AUDIO_PRESENTATION_DELAY_US))
+
+#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE                                           \
+	BT_AUDIO_LC3_UNICAST_PRESET_SOURCE_RECOMMENDED(BT_AUDIO_LOCATION_FRONT_LEFT,               \
+						       BT_AUDIO_CONTEXT_TYPE_MEDIA)
 
 #if CONFIG_BT_AUDIO_UNICAST_RECOMMENDED
-#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO                                                  \
-	BT_AUDIO_LC3_UNICAST_PRESET_RECOMMENDED(BT_AUDIO_LOCATION_FRONT_LEFT,                      \
-						BT_AUDIO_CONTEXT_TYPE_MEDIA)
+#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                             \
+	BT_AUDIO_LC3_UNICAST_PRESET_SINK_RECOMMENDED(BT_AUDIO_LOCATION_FRONT_LEFT,                 \
+						     BT_AUDIO_CONTEXT_TYPE_MEDIA)
 
 #elif CONFIG_BT_AUDIO_UNICAST_CONFIGURABLE
-#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO                                                  \
+#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                             \
 	BT_AUDIO_LC3_PRESET_CONFIGURABLE(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
 
 #elif CONFIG_BT_AUDIO_UNICAST_16_2_1
-#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO                                                  \
+#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                             \
 	BT_AUDIO_LC3_UNICAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT,                           \
 					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
 
 #elif CONFIG_BT_AUDIO_UNICAST_24_2_1
-#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO                                                  \
+#define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                             \
```

Comment:
What if we want to set UNICAST_24_2_1 for both sink and source?

Reply:
Changed so that both sink and source use the same preset